src/test/java/com/gaejangmo/apiserver/model/product/service/ProductServiceTest.java

```diff
@@ -0,0 +1,43 @@
+package com.gaejangmo.apiserver.model.product.service;
+
+import com.gaejangmo.apiserver.model.product.domain.ProductRepository;
+import com.gaejangmo.apiserver.model.product.domain.ProductTestData;
+import com.gaejangmo.apiserver.model.product.domain.vo.ProductName;
+import com.gaejangmo.apiserver.model.product.dto.ProductResponseDto;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+
+@ExtendWith(SpringExtension.class)
+class ProductServiceTest {
+    private static final String TARGET = "애플 맥북 프로 15형 2019년형 MV912KH/A";
```

Comment:
```suggestion
    private final static String PRODUCT_NAME = "애플 맥북 프로 15형 2019년형 MV912KH/A";
```
`TARGET` 보다는 `PRODUCT_NAME` 같은게 좀 더 명시적일거 같아요.